inst/tests/test-annotate.r

```diff
@@ -0,0 +1,31 @@
+context("annotate")
+
+test_that("dates in segment annotation work", {
+  dt <-
```

Comment:
Could you please simplify this dataset to just use a couple of dates created by hand?